package.json

```diff
@@ -30,7 +30,7 @@
     "@types/chai": "4.2.0",
     "@types/chai-as-promised": "^7.1.2",
     "@types/mocha": "^5.2.7",
-    "@types/node": "^12.12.21",
+    "@types/node": "^14.0.27",
```

Comment:
why? we aren't using node14

Reply:
I bumped because a dependency uses incompatible types that where not causing issues because strictNullChecks was false. A github issue about recommended to bump `@types/node`. I didn't knew the `@types/node` semver was related to NodeJS version.

Reply:
Since I'm not turning `strictNullChecks` on yet, I'll revert the version bump and we can deal with the dependency issue latter.